### PR TITLE
Fix typos in documentation

### PR DESCRIPTION
**Pull request by leonardoalt**

### CODE REVIEW

File: zokrates_book/src/sha256example.md [hunk 1]
```diff
@@ -52,8 +52,8 @@ grep '~out' witness
 which should lead to the following output:
 
 ```sh
-~out_0 263561599766550617289250058199814760685
 ~out_1 65303172752238645975888084098459749904
+~out_0 263561599766550617289250058199814760685
```
Comment (leonardoalt) [inline zokrates_book/src/sha256example.md]: This is the order I got when running the command.

File: zokrates_book/src/concepts/imports.md [hunk 2]
```diff
@@ -38,10 +38,10 @@ from "./path/to/my/module" import main as module
 Note that this legacy method is likely to be become deprecated, so it is recommended to use the preferred way instead.
 ### Symbols
 
-Two type of symbols can be imported
+Two type of symbols can be imported:
```
Comment (Schaeff) [inline zokrates_book/src/concepts/imports.md]:
```suggestion
Two types of symbol can be imported:
```

is this correct english

File: zokrates_book/src/sha256example.md (hunk 1 above)
Comment (Schaeff) [reply zokrates_book/src/sha256example.md]: Interesting. We need to investigate the cause of this

File: zokrates_book/src/concepts/imports.md (hunk 2 above)
Comment (leonardoalt) [reply zokrates_book/src/concepts/imports.md]: According to https://english.stackexchange.com/questions/5539/types-of-things-vs-types-of-thing
I think it should be `types of symbols`

File: zokrates_book/src/sha256example.md (hunk 1 above)
Comment (leonardoalt) [reply zokrates_book/src/sha256example.md]: Do you get the original doc version if you run it? I can remove this change for now while we investigate.

File: zokrates_book/src/concepts/imports.md (hunk 2 above)
Comment (Schaeff) [reply zokrates_book/src/concepts/imports.md]: fair enough
```suggestion
Two types of symbols can be imported:
```

Comment (leonardoalt) [reply zokrates_book/src/concepts/imports.md]: That's how it is right now

**Pull request merged**